generator/tools/helpers.js

```diff
@@ -32,6 +32,23 @@ function updateBabelConfig (api, callback) {
   }
 }
 
+function updateTypeScriptConfig (api, invoking) {
+  // inject vuetify types to tsconfig.json
+  if (invoking) {
+    api.render(files => {
+      const tsconfig = files['tsconfig.json']
+      if (tsconfig) {
```

Comment:
imho it should create tsconfig.json if the file does not exist

Reply:
Wouldn't this exist if the vue project was created with typescript support.  If it doesn't exist then presumably there is no TS support in the project.